bin/detect

```diff
@@ -6,9 +6,11 @@ build=$(cd "$1/" && pwd)
 
 if test -f "$build/Godeps/Godeps.json" || # godeps
    test -f "$build/vendor/vendor.json" || # govendor
-   (test -d "$build/src" && test -n "$(find "$build/src" -mindepth 2 -type f -name '*.go' | sed 1q)") # gb
+   (test -d "$build/src" && test -n "$(find "$build/src" -mindepth 2 -type f -name '*.go' | sed 1q)") || # gb
+   test -f "$build/main.go" ||
+   test -f "$build/cmd/main.go"
```

Comment:
This would make the executable named "cmd". It likely that we want to check for `$build/cmd/**/main.go` instead?